cardano-cli/src/Cardano/CLI/Read.hs

```diff
@@ -244,7 +244,7 @@ renderScriptWitnessError = \case
     pretty file <> ": expected a script in the Plutus script language, " <>
     "but it is actually using " <> pshow lang <> "."
   ScriptWitnessErrorReferenceScriptsNotSupportedInEra anyEra ->
-    "Reference scripts not supported in era: " <> pretty anyEra
+    "Reference scripts not supported in era: " <> pshow anyEra
```

Comment:
I think we should provide `Pretty AnyShelleyBasedEra` similarly to `Pretty AnyCardanoEra` and just use `pretty` instead of `pshow`.